vanilla-sms-app/index.html

```diff
@@ -36,11 +36,9 @@
 	<script src="libs/evothings/evothings.js"></script>
 	<script>
 	var app = {
-    	sendSms: function() {
+		sendSms: function() {
 			var number = document.getElementById('numberTxt').value;
 			var message = document.getElementById('messageTxt').value;
-			alert(number);
-			alert(message);
 
 			//CONFIGURATION
```

Comment:
Suggestion: clarify why each option is needed and how they can be used. Check whether options can be omitted for having more straightforward code.